python/amici/petab_import.py

```diff
@@ -311,12 +332,11 @@ def import_model(sbml_model: Union[str, 'libsbml.Model'],
         sigmas=sigmas,
         allow_reinit_fixpar_initcond=allow_reinit_fixpar_initcond,
         noise_distributions=noise_distrs,
+        verbose=verbose,
         **kwargs)
     end = time. time()
 
-    if verbose:
-        logger.log(logging.INFO, f"{Fore.GREEN}Model imported successfully in "
-                                 f"{round(end - start, 2)}s")
+    logger.info(f"Model imported successfully in {round(end - start, 2)}s")
```

Comment:
use https://github.com/ICB-DCM/AMICI/blob/a0407673453d6e18a9abec5b6f73758dd09f7aaf/python/amici/logging.py#L139 ?

Reply:
Will do. Thx for reminder.